### PR TITLE
Update clause_2.adoc

### PR DESCRIPTION
**Pull request by FransKnibbe**
Assorted small textual changes.

### CODE REVIEW

File: white_paper/clause_2.adoc [hunk 1]
```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
...
-==== Beneficiary 1: Data Consumers
+==== Beneficiary 1: Data consumers
 
-Data consumers use data, including geospatial data. They key needs are:
+Data consumers use data, including geospatial data. Their key needs are:
 
-1. to access and understand data from multiple sources; and
-2. to integrate data - geospatial and non-geospatial
-3. to get a data quality evaluation of the given geospatial data
+1. to access and understand data from multiple sources,
```
Comment (jabhay) [inline white_paper/clause_2.adoc]:
```suggestion
1. to access and understand data from multiple sources;
```

File: white_paper/clause_2.adoc [hunk 2]
```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
...
+2. to integrate data - geospatial and non-geospatial, and
```
Comment (jabhay) [inline white_paper/clause_2.adoc]:
```suggestion
2. to integrate data - geospatial and non-geospatial; and
```

File: white_paper/clause_2.adoc [hunk 3]
```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
...
-==== Beneficiary 2: Data Publishers
+==== Beneficiary 2: Data publishers
 
 Data publishers have data, including geospatial data. Their key needs are:
 
-1. to maximise use of their data; and
-2. to manage their data as simply as possible
-3. to indicate possible usage contexts of the geospatial data
-4. to quality-assure their geospatial data
+1. to maximise use of their data,
```
Comment (jabhay) [inline white_paper/clause_2.adoc]:
```suggestion
1. to maximise use of their data;
```

File: white_paper/clause_2.adoc [hunk 4]
```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
...
+2. to manage their data as simply as possible,
```
Comment (jabhay) [inline white_paper/clause_2.adoc:20]:
```suggestion
2. to manage their data as simply as possible;
```

File: white_paper/clause_2.adoc [hunk 5]
```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
...
+3. to indicate possible usage contexts of the geospatial data, and
```
Comment (jabhay) [inline white_paper/clause_2.adoc]:
```suggestion
3. to indicate possible usage contexts of the geospatial data; and
```

**Review by jabhay — CHANGES_REQUESTED** [review]
I'd like to stick to the convention of using semi colons in lists. If you strongly disagree, let me know and I'll accept as is

File: white_paper/clause_2.adoc (hunk 5 above)
Comment (FransKnibbe) [reply white_paper/clause_2.adoc]: I wondered about conventions for punctuation for list in English and the first hit I got on Google showed the method I uses. But this page seems authoritative enough, and it demonstrates using semicolons: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html. So yeah, that seems right.

**Comment by FransKnibbe** [comment]
superseded by newer pull request

**Pull request closed without merging**